src/main/java/at/hannibal2/skyhanni/utils/SkyBlockItemModifierUtils.kt

```diff
@@ -242,6 +242,12 @@ object SkyBlockItemModifierUtils {
     //$$ fun ItemStack.getMinecraftId() = Registries.ITEM.getId(item)
     //#endif
 
+    //#if MC < 1.21
+    fun String.isVanillaItem() = Item.itemRegistry.getObject(ResourceLocation(this)) != null
```

Comment:
do not use public extended functions for generic objects. in this case, we dont want the funciton isVanillaItem as suggestion for every string in skyhanni. please use a normal function instead